covid19-notebooks/covid19-notebook-etl-run-with-slack.sh

```diff
@@ -0,0 +1,16 @@
+if ! bash "/nb-etl/covid19-notebook-etl-run.sh"; then
```

Comment:
looks good just wondering why you have the fail state before the success here

Reply:
oh honestly not sure, i just copied the file Jawad wrote 😄  https://github.com/uc-cdis/covid19-tools/pull/184/files